src/XRP/model/xrp-signer.ts

```diff
@@ -15,10 +16,28 @@ export default class XrpSigner {
    */
   public static from(signer: Signer): XrpSigner | undefined {
     const account = signer.getAccount()?.getValue()?.getAddress()
```

Comment:
Does this need to be converted to an `XAddress`?

Reply:
good catch, yeah let's do that.  SDK philosophy is that all addresses that appear in SDK types should be X-Address encoded.  Be sure to update the description for the `account` param in the main constructor.  I think just adding ", encoded as an X-Address." to the end of the existing description should do.

I guess you'll probably have to update the existing test too.